statefun-flink/statefun-flink-datastream/pom.xml

```diff
@@ -76,12 +76,6 @@ under the License.
             <version>${flink.version}</version>
             <scope>provided</scope>
         </dependency>
-        <dependency>
-            <groupId>org.apache.flink</groupId>
-            <artifactId>flink-java</artifactId>
-            <version>${flink.version}</version>
-            <scope>provided</scope>
-        </dependency>
```

Comment:
As far as I can tell, this dependency was added by accident. Let me know if I should revert this commit.

Reply:
also curious why the dependency on `statefun-flink-io`?